setup.py

```diff
@@ -28,10 +27,9 @@ def read(*rnames):
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
         'Framework :: Django',
         'Topic :: Database'
     ],
-    install_requires=['django-bulk-update==1.1.10']
+    install_requires=['django-bulk-update==2.2.0']
```

Comment:
As this is a separate standalone project, add Django to `install_requires` and remove requirements.txt
https://packaging.python.org/discussions/install-requires-vs-requirements/